cmd/kubepkg/templates/latest/deb/kubectl/debian/changelog

```diff
@@ -1,6 +1,5 @@
 kubectl ({{ .Version }}-{{ .Revision }}) {{ .Channel }}; urgency=medium
 
-  * https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md
+  * https://git.k8s.io/kubernetes/CHANGELOG/README.md
 
  -- Kubernetes Authors <kubernetes-dev@googlegroups.com>  {{ date }}
-
```

Comment:
Is removing these trailing newlines intentional?

Reply:
This was my editor :) So yes, since it does not seems to be needed :innocent:

Reply:
Gotcha! I am guessing it doesn't affect anything, but I am always hesitant when seeing this sort of thing in case there is a reason the newlines exist :)

Reply:
@saschagrunert -- Which editor are you using? Would prefer if we didn't change the newlines.

Reply:
Vim, but yeah I can revert it :+1: